examples/src/main/python/ml/simple_params_example.py

```diff
@@ -70,18 +71,18 @@
 
     # We may alternatively specify parameters using a parameter map.
     # paramMap overrides all lr parameters set earlier.
-    paramMap = {lr.maxIter: 20, lr.thresholds: [0.45, 0.55], lr.probabilityCol: "myProbability"}
+    paramMap = {lr.maxIter: 20, lr.thresholds: [0.5, 0.5], lr.probabilityCol: "myProbability"}
```

Comment:
I think it's not necessary to update here. If users explicitly set ```thresholds``` then ```threshold``` will be cleared and takes no effect, and vice verse. In this example, users explicitly set ```thresholds``` to ```[0.45, 0.55]``` is okey and will not throw any exception.